deploy-manage/deploy/cloud-enterprise/ece-architecture.md

```diff
@@ -86,3 +86,24 @@ Services are deployed as Docker containers, which simplifies the operational eff
 
     Docker containers communicate securely with one another through Transport Layer Security, provided by [Stunnel](https://www.stunnel.org/) (as not all of the services or components support TLS natively). Tunneling all traffic between containers makes sure that it is not possible to eavesdrop, even when someone else has access to the underlying cloud or network infrastructure.
 
+## ECE service containers per host role(s) [ece-service-containers]
+
+Each {{ece}} service runs as a dedicated container. These containers are automatically deployed based on the roles assigned to each ECE host. The following table lists the containers you’ll find on your ECE hosts:
+
+| Container                                          | Host roles         | Description |
+|---|---|---|
+| `frc-beats-runners-beats-runner`                  | All roles     | Collects metrics from local containers for monitoring and health checks. |
+| `frc-client-forwarders-client-forwarder`          | All roles     | Manages communication between hosts and ZooKeeper. |
+| `frc-runners-runner`                              | All roles     | Runs on every ECE host and provides a supervisor service to deploy and manage containers according to their defined roles, ensuring they are online and healthy. |
```

Comment:
`according to their defined roles` - this suggests the _containers_ have roles defined, whereas it's _hosts_ the roles pertain to. 
In other words, a host has a role assigned to it and that role defines what containers should run on the host. 

The runner does indeed manage these containers, makes sure they're started at the proper version, etc.  , but saying that it `ensures they are online and healthy` isn't very accurate. I'd rephrase this whole line to sth like:
```
Runs on every ECE host and provides a supervisor service to deploy and manage containers based on the host's role. Ensures the required containers are started at the proper version.
```

Reply:
done